app/_src/gateway-operator/production/upgrade/data-plane/blue-green.md

```diff
@@ -0,0 +1,173 @@
+            },
+            "services": {
+                "adminAPI": {
+                    "addresses": [
+                        {
+                            "sourceType": "PrivateIP",
+                            "type": "IPAddress",
+                            "value": "None"
+                        }
+                    ],
+                    "name": "dataplane-admin-bluegreen-cx6nq"
+                },
+                "ingress": {
+                    "addresses": [
+                        {
+                            "sourceType": "PrivateLoadBalancer",
+                            "type": "IPAddress",
+                            "value": "172.18.0.101"
+                        },
+                        {
+                            "sourceType": "PrivateIP",
+                            "type": "IPAddress",
+                            "value": "10.96.28.2"
+                        }
+                    ],
+                    "name": "dataplane-ingress-bluegreen-2249g"
+                }
+            }
+        }
+    ]
+    ```
+
+    Here you can see the ingress `Service` name that was created for you to validate the new set of `Pod`s.
+    Its name is stored in `status.rollout.service`.
+    For convenience its addresses (together with their `type`s and `sourceType`s) are stored `status.rollout.services[].ingress`.
+
+    Notice that `status.rollout.condtition` contains a condition with Type `RolledOut` and `Reason` set to `AwaitingPromotion`.
+    This means that everything is ready to promote the `DataPlane` with staged changes.
+    Before we do so, let's test it.
+
+1. You can access the spawned "preview" `Service` by using its LB address (taken from the `status.rollout.services.ingress` field):
+
+    ```bash
+    $ curl -s -D - -o /dev/null 172.18.0.101
+    HTTP/1.1 404 Not Found
+    Date: Thu, 21 Sep 2023 11:40:26 GMT
+    Content-Type: application/json; charset=utf-8
+    Connection: keep-alive
+    Content-Length: 52
+    X-Kong-Response-Latency: 0
+    Server: kong/3.3.1
+    ```
+
+    This way we can see that new {{ site.base_gateway }} `Pod`s have been deployed and are reachable through the "preview" ingress `Service` using the updated image.
+
+1. Verify the old `Pod`s are still available and are still serving the traffic. You can verify that by accessing its "live" ingress `Service`
+
+    We can get its addresses with:
+
+    ```bash
+    $ kubectl get dataplane bluegreen -o jsonpath-as-json='{.status.addreses}
+    [
+      [
+        {
+            "sourceType": "PrivateLoadBalancer",
+            "type": "IPAddress",
+            "value": "172.18.0.100"
+        },
+        {
+            "sourceType": "PrivateIP",
+            "type": "IPAddress",
+            "value": "10.96.11.156"
+        }
+      ],
+    ]
+    ```
+
+    and issue a request:
+
+    ```bash
+    $ curl -s -D - -o /dev/null 172.18.0.100
+    HTTP/1.1 404 Not Found
```

Comment:
https://datatracker.ietf.org/doc/html/rfc5737.

Reply:
This is fine. It's just to showcase that this IP is indeed what we got in `DataPlane` `status`.

One thing we might consider is explaining that this 404 is expected (Kong Gateway with no route will return just that).